### DISCUSSION TITLE
Enable window controls?

### DISCUSSION THREAD

**Discussion by moontowncitizen** [adriankarlen/textfox#75] [Q&A] [answered]
how could I add some css to create another box in the top right, I'm pretty new to Firefox theming and have limited knowledge on css. Here is kind of what I want, I want to make another .css file and import that into the userChrome.css
``#window-controls-container {
  position: absolute;
  top: 0;
  right: 0;
  display: flex;
  align-items: center;
  margin: 8px;
  padding: 4px;
  border: var(--border-width) solid var(--tf-border);
  border-radius: var(--tf-rounding);
  background-color: var(--tf-bg);
  z-index: 10;
  transition: border-color var(--tf-border-transition);
}

#window-controls-container:hover {
  border-color: var(--tf-accent);
}

/* Individual buttons */
#window-controls-container button {
  appearance: none;
  background: none;
  border: none;
  margin: 0 2px;
  padding: 4px;
  color: var(--text-color);
  cursor: pointer;
  font-size: 1rem;
  transition: color 0.2s, background-color 0.2s;
}

#window-controls-container button:hover {
  background-color: var(--tf-accent);
  color: var(--tf-bg);
  border-radius: 4px;
}

/* Specific button icons */
#window-controls-minimize::before {
  content: "-";
}

#window-controls-maximize::before {
  content: "□";
}

#window-controls-close::before {
  content: "×";
  color: var(--error-color); /* For a distinctive close button */
}

#window-controls-close:hover {
  background-color: var(--error-color);
  color: var(--tf-bg);
}
`

**Comment by moontowncitizen** [comment]
I want to get some rounded edges around the whole window I just have no clue how to do that maybe with body?

**Comment by adriankarlen** [comment, marked as answer]
https://github.com/adriankarlen/textfox/pull/90 adds configurable window controls display state when using native vertical tabs or horizontal tabs.

If with the whole body you mean the rounding of the blocks there is a variable `--tf-rounding` that can be used.
